dietpi/dietpi-software

```diff
@@ -1073,6 +1073,15 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_DEPS[$software_id]='188'
 		(( $G_HW_ARCH == 10 )) || aSOFTWARE_DEPS[$software_id]+=' 16'
 		aSOFTWARE_INTERACTIVE[$software_id]=1
+		#------------------
+		software_id=125
+
+		aSOFTWARE_NAME[$software_id]='Synapse Server'
```

Comment:
Shall we name it simply "Synapse"? There is no Synapse client, but general chat clients, like Element, so I think it's sufficient.